### ISSUE TITLE
BarkTTS doesn't work

### ISSUE THREAD

**Issue by ccateni** [IAHispano/Applio-RVC-Fork#19]
It keeps giving me an error.

loading weights\Sonic48k.pth
gin_channels: 256 self.spk_embed_dim: 109
<All keys matched successfully>
['This is a test audio.', 'I hope this sounds good...']
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:10000 for open-end generation.
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:10000 for open-end generation.
Error opening 'F:\\downloads\\Applio-RVC-Fork\\audio-outputs\\bark_out.wav': System error.

**Comment by blaise-tk** [comment]
TTS Locally doesen't work properly

**Comment by ccateni** [comment]
> TTS Locally doesen't work properly

Do you know whats causing the issue for local versions and when it maybe fixed?

**Comment by ccateni** [comment]
So it's fixed in local?
